### ISSUE TITLE
[Bug]: UI is stuck and not working.

### ISSUE THREAD

**Issue by yukiyogit** [lllyasviel/stable-diffusion-webui-forge#89]
### Checklist

- [ ] The issue exists after disabling all extensions
- [X] The issue exists on a clean installation of webui
- [ ] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [ ] The issue exists in the current version of the webui
- [ ] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/67177866/15171a60-8643-4148-8441-f3d94f894fcf)

When editing the prompt, the image generation button suddenly says "Interrupting...". The UI stops working. It does not come back and nothing can be done.

I have to restart the UI to fix the problem.

### Steps to reproduce the problem

Frequently occurs when typing prompts

### What should have happened?

It's odd that the UI displays "Interrupting..." while typing prompts. " and the UI freezes, I want to generate an image so the UI doesn't freeze.

### What browsers do you use to access the UI ?

Google Chrome

### Sysinfo

[sysinfo-2024-02-07-03-35.json](https://github.com/lllyasviel/stable-diffusion-webui-forge/files/14188719/sysinfo-2024-02-07-03-35.json)

### Console logs

```Shell
To load target model SDXLClipModel
Begin to load 1 model
Moving model(s) has taken 1.13 seconds
Model loaded in 17.4s (unload existing model: 1.8s, calculate hash: 8.7s, load weights from disk: 0.1s, forge load real models: 4.5s, calculate empty prompt: 2.2s).
To load target model SDXLClipModel
Begin to load 1 model
unload clone 0
Moving model(s) has taken 1.25 seconds
To load target model SDXL
Begin to load 1 model
Moving model(s) has taken 1.68 seconds
100%|██████████████████████████████████████████| 30/30 [00:04<00:00,  6.35it/s]
To load target model AutoencoderKL█████████████| 30/30 [00:04<00:00,  6.63it/s]
Begin to load 1 model

0: 640x512 2 faces, 127.0ms
Speed: 2.5ms preprocess, 127.0ms inference, 1.5ms postprocess per image at shape (1, 3, 640, 512)
100%|██████████████████████████████████████████| 13/13 [00:02<00:00,  6.17it/s]
100%|██████████████████████████████████████████| 13/13 [00:02<00:00,  6.09it/s]
Total progress: 100%|██████████████████████████| 30/30 [00:11<00:00,  2.60it/s]
Total progress: 100%|██████████████████████████| 30/30 [00:11<00:00,  6.63it/s]
```

**Comment by yukiyogit** [comment]
It is a very serious bug and does not generate images at all. This makes it unusable. This is a bug that was not in the original web ui.

**Comment by PureUnadulteratedEgo** [comment]
I have to agree that this is a serious bug and issue that's not present in the original web-ui. Luckily, I've managed to be able to reproduce this:

1) Press the escape key while you're editing the prompt or searching for a Lora (it has happened to me in both contexts).
2) The UI seems to try to interrupt something and gets stuck as @sicatt mentioned.

It's really an incredible UI so far, but unless a fix or workaround is found... this makes it basically unusable. You're always a single keystroke away from having to reload the entire UI.

**Comment by huchenlei** [comment]
This feature comes from upstream A1111 repo here: https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/1d42babd324b933bae317cb427fe0513138954f4 

Do you notice the same behaviour in original A1111?

**Comment by PureUnadulteratedEgo** [comment]
I'll try to clone the dev branch of A1111 as I don't want to fuck up my main install. Other people were able to recreate the issue by just using the escape key in the prompt.

**Comment by PureUnadulteratedEgo** [comment]
@huchenlei I was indeed able to recreate it on the dev branch of A1111.

**Comment by PureUnadulteratedEgo** [comment]
If anybody else is having a problem, my current workaround is just directly editing the line 165 in script.js.

`    if (isEsc) {`
For
`    if (isAltKey && isCtrlKey && isEnter) {`
